tests/unit/test_worker.py

```diff
@@ -668,6 +671,28 @@ def task_func():
     assert job_row["status"] == expected_status
     assert job_row["attempts"] == expected_attempts
 
+    assert (
+        len(
+            [
+                record
+                for record in caplog.records
+                if record.levelname == "INFO" and "to retry" in record.message
+            ]
+        )
+        == 1
+    )
```

Comment:
I'm a believe that the `==1` line produced by the formatter is an indication that the code would be more readable with an intermediary variable=
```python
info_records = [...]
assert len(info_records) == 1
```

Usually in those case, the code ends up being more readable AND less lines :)
(same below)